### PR TITLE
add quick vocab list

### PR DESCRIPTION
**Pull request by dcreemer**

### CODE REVIEW

File: vocab.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+count = 23
+```
+
+### Comment
+Non-functioning text in your program, after the '#'
+
+#### example:
+```python
+x = x + 10.0    # move 10 pixels to the right
+```
```
Comment (annalizjensen) [inline vocab.md]: can you include multi-line comments here as well?

**Review by annalizjensen — APPROVED** [review]
just one minor comment

**Comment by dcreemer** [comment]
added multi-line example

**Comment by annalizjensen** [comment]
cool +1, we can add to this as we cover more chapters. conditions, boolean operators, comparison operators, control flow all probably need to be added, but we can do that later.

**Comment by annalizjensen** [comment]
@dcreemer i'm merging this fyi

**Pull request merged**